exampleCourse/questions/examplesRAddTwoNumbers/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "69994f6e-0150-4c29-bc6f-87017719f7b1",
+    "title": "Add Two Numbers through R",
+    "topic": "R",
+    "tags": ["numeric", "balamut2", "v3", "Fa19", "rstats"],
+    "type": "v3"
+}
```

Comment:
nit: all the JSON files are missing a newline at the end of file. It’s no big deal, but it makes it weird to cat a file or things like that. The Python files are all fine.